recipe3.html

```diff
@@ -0,0 +1,64 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Milestone1</title>
+    <link rel="stylesheet" href="index.css">
+</head>
+<body>
+    <!--recipe page header-->
+    <div class="header">
+        <a class="logo" href="index.html">
+            <h1>Angel Cooks</h1>
+        </a>
+        <nav>
+            <a class="nav-links" href="index.html">Home</a>
+            <a class="nav-links" href="about.html">About Me</a>
+        </nav>
+    </div>
+    <!--recipe page content-->
+    <main>
+        <div class="pg-content">
+            <!--recipe page card-->
+            <div class="pg-cards">
+                <img class="card-img" src="https://alittlebityummy.com/wp-content/uploads/2021/05/Low-FODMAP-Sticky-Garlic-Tofu-Veggie-Bowl.jpg.webp">
+                <div class="card-body">
+                    <button class="recipe-btn">Garlic Tofu Veggie Bowl</button>
+                    <p>With a tummy-friendly garlic infused oil, enjoy garlic in your low FODMAP diet!</p>
+                </div>
+            </div>
```

Comment:
FYI the content for this div does not always fit the div on wider screens. Try changing the height property for `pg-cards` so it scales with the content
![image](https://user-images.githubusercontent.com/32073638/199340535-4ba3e63c-c95f-49ab-8814-7302cd52482e.png)